maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -23972,9 +23972,7 @@
 	},
 /area/almayer/living/cryo_cells)
 "bAZ" = (
-/obj/structure/machinery/cm_vending/clothing/vehicle_crew{
-	pixel_x = 32
-	},
+/turf/open/space/basic,
```

Comment:
what's going on here? it seems to be the exact same as when nanu remarked on it

Reply:
I have no idea, thought I offset it